### ISSUE TITLE
What's the proper way to use "load_css()"?

### ISSUE THREAD

**Issue by LOL-XDD** [r0x0r/pywebview#252]
### Specification
- Platform: Windows 10
- Version: 2.0.3

### Description

The command I'm using is "webview.load_css('body {background: red!important;', uid='master')". I'm using it as a test command to load the css into my webpage, and I'm getting the following error:

```
Traceback (most recent call last):
  File "C:\Users\Owner\AppData\Local\Programs\Python\Python35-32\lib\site-packages\webview\__init__.py", line 132, in wrapper
    return function(*args, **kwargs)
  File "C:\Users\Owner\AppData\Local\Programs\Python\Python35-32\lib\site-packages\webview\__init__.py", line 286, in evaluate_js
    return gui.evaluate_js(escaped_script, uid)
  File "C:\Users\Owner\AppData\Local\Programs\Python\Python35-32\lib\site-packages\webview\winforms.py", line 375, in evaluate_js
    window = BrowserView.instances[uid]
KeyError: 'master'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 4, in <module>
    webview.evaluate_js(script, uid='master')
  File "C:\Users\Owner\AppData\Local\Programs\Python\Python35-32\lib\site-packages\webview\__init__.py", line 141, in wrapper
    raise Exception('Cannot call function: No webview exists with uid: {}'.format(uid))
Exception: Cannot call function: No webview exists with uid: master
```

**Comment by r0x0r** [comment]
Could not reproduce this on Windows. I have used

```
import webview
import threading

"""
This example demonstrates how to load HTML in a web view window
"""


def load_html():
    webview.load_html('<h1>This is dynamically loaded HTML</h1>')
    webview.load_css('body {background: red}')


if __name__ == '__main__':
    t = threading.Thread(target=load_html)
    t.start()

    webview.create_window('Load HTML Example')
```

**Comment by LOL-XDD** [comment]
@r0x0r @fordffx 

None of the above examples work.

    import webview
    import threading

    def css_inject():
        webview.load_css('body {background: red}')

    if __name__ == '__main__':
        t = threading.Thread(target=css_inject)
        t.start()
        webview.create_window("Midlii", "https://www.vidlii.com", width=275, height=467)

I tried both exactly as they were typed in the window and both gave me the same white screen with this error;

        Exception in thread Thread-1:
    Traceback (most recent call last):
      File "C:\Users\Owner\AppData\Local\Programs\Python\Python35-32\lib\threading.py", line 914, in _bootstrap_inner
        self.run()
      File "C:\Users\Owner\AppData\Local\Programs\Python\Python35-32\lib\threading.py", line 862, in run
        self._target(*self._args, **self._kwargs)
      File "main.py", line 11, in load_html
        webview.load_css('body {background: red}')
      File "C:\Users\Owner\AppData\Local\Programs\Python\Python35-32\lib\site-packages\webview\__init__.py", line 132, in wrapper
        return function(*args, **kwargs)
      File "C:\Users\Owner\AppData\Local\Programs\Python\Python35-32\lib\site-packages\webview\__init__.py", line 239, in load_css
        gui.evaluate_js(code)
    TypeError: evaluate_js() missing 1 required positional argument: 'uid'

**Comment by fordffx** [comment]
replace these below:
--------------------------
webview.load_html('<h>hello master window</h>')
webview.load_css('body {background: red!important;', uid='master')
---------------------------
with these below:
-----------------
webview.load_html('<style type="text/css">body {background: red!important;</style><h>hello master window</h>')
-----------------
if u want to change the page color dynamicly,
===========
backcolor=XXX
threading.Thread(target=kkk, args=(backcolor))
===========
and pass backcolor value to 
=================
webview.load_html('<style type="text/css">body {background:'+backcolor +';</style><h>hello master window</h>')
let html engine to parse it

**Comment by r0x0r** [comment]
@LOL-XDD This is a bug in 2.0.3. Fixed in master some time ago.

**Comment by r0x0r** [comment]
Fixed in 2.1
